post-replies.js

```diff
@@ -17,7 +17,7 @@ class PostReplies extends HTMLElement {
       // Check if the note has a replies collection and use it if available
       const note = await db.getNote(postUrl)
```

Comment:
Instead we should ingest the note and do a `searchNotes({inReplyTo: noteURL})` to get the list